assets/translations/nb-NO.json

```
@@ -93,19 +87,25 @@
     "details": "Produktdetaljer",
     "select": "Velg produkter",
     "errorLoading": "Feil ved lasting av produkter",
     "noProductsFound": "Ingen produkter funnet",
     "addFirst": "Legg til ditt første produkt for å komme i gang",
-    "settingsSoon": "Innstillingsfunksjon kommer snart!",
-    "addSoon": "Legg til produkt-funksjon kommer snart!",
     "add": "Legg til produkt",
+    "edit": "Rediger produkt",
+    "name": "Produktnavn",
+    "name_required": "Vennligst oppgi produktnavn",
+    "description": "Beskrivelse",
```

Biome (2.4.4)

[error] The key description was already declared.

(lint/suspicious/noDuplicateObjectKeys)